src/containers/Accounts/AccountTransactionTable/AccountTransactionTable.tsx

```diff
@@ -10,12 +10,14 @@ import { ANALYTIC_TYPES, analytics } from '../../shared/utils'
 
 export interface AccountTransactionsTableProps {
   accountId: string
-  currencySelected: string
+  currencySelected?: string
+  hasTokensColumn: boolean
 }
 
 export const AccountTransactionTable = ({
   accountId,
-  currencySelected,
+  currencySelected = 'XRP',
+  hasTokensColumn,
```

Comment:
It's bad practice to have an optional param before a required param.

Reply:
done